build/templates/grpc_session_options.py.mako

```diff
@@ -41,24 +45,34 @@ class SessionInitializationBehavior(IntEnum):
 class GrpcSessionOptions(object):
     '''Collection of options that specifies session behaviors related to gRPC.'''
 
-    def __init__(self, grpc_channel, session_name, initialization_behavior=SessionInitializationBehavior.AUTO):
+    def __init__(
+        self,
+        grpc_channel,
+        session_name,
+        *,
+        api_key=MEASUREMENTLINK_23Q1_NIMI_PYTHON_API_KEY,
+        initialization_behavior=SessionInitializationBehavior.AUTO
```

Comment:
What's the point of specifying default values if these are going to be keyword-only arguments?

Reply:
You *must* specify default values for named keyword-only arguments. Or are you suggesting using a default of `None`? For the `None` case, you still have to turn it into the desired value, and it's just easier to specify the desired value as the default instead of having to explicitly check for `None`.

Reply:
Oh, I somehow convinced myself keyword-only meant they wouldn't be used if not passed. Nevermind.

Reply:
Specifying the desired value also informs type-checking tools like mypy and IDEs like VSCode what type to expect.